script.js

```diff
@@ -0,0 +1,38 @@
+
+const postApi = "https://jsonplaceholder.typicode.com/todos";
+
+// Fetch API Function
+async function fetchTodo(apiUrl) {
+    try {
+        const response = await fetch(apiUrl);
+        if (!response.ok) {
+            throw new Error('Network response was not ok');
+        }
+        const json = await response.json();
+        return json;
+    } catch (error) {
+        console.error('There was a problem with the fetch operation:', error);
+    }
+}
+
+
+// Append data in DOM 
+async function appendData(data) {
```

Comment:
Why is this function asynchronous?

Reply:
I thought this will wait to load data from fetch, that's why